wp-multi-network/includes/classes/class-wp-ms-network-command.php

```diff
@@ -11,7 +11,7 @@
  *
  * @since 1.3.0
  */
-class WP_MS_Network_Command extends WP_CLI_Command {
```

Comment:
Note: Historically, WP-CLI provided a base WP_CLI_Command class to extend, however extending this class is not required and will not change how your command behaves.

https://make.wordpress.org/cli/handbook/guides/commands-cookbook/